salt/minion.sls

```diff
@@ -55,14 +54,16 @@ salt-minion-macos:
       - file: salt-minion-macos
     - require_in:
       - service: salt-minion
+    {%- endif %}
+{%- endif %}
 
 salt-minion:
 {% if salt_settings.install_packages %}
   {%- if grains.os == 'MacOS' and salt_settings.salt_minion_pkg_source %}
   macpackage.installed:
     - name: '/tmp/salt.pkg'
     - target: /
-    {# macpackage.installed behaves weirdly with version_check; version_check detects difference but fails to actually complete install. #}
+    {# macpackage.installed behaves weirdly with version_check, decects diff but fails to complete install. #}
```

Comment:
s/decects/detects/ maybe?

Reply:
Thanks, I'm trying to see if CI will work (commits are gone a bit funny). But anyway will see ...